var/spack/repos/builtin/packages/rivet/package.py

```diff
@@ -280,6 +283,7 @@ class Rivet(AutotoolsPackage):
     depends_on("yaml-cpp", when="@2.0.0:2.1.2")
 
     depends_on("autoconf", type="build")
+    depends_on("autoconf@2.71:", when="@3.1.7", type="build")
```

Comment:
any particular reason for this lowerbound?

Reply:
https://gitlab.com/hepcedar/rivet/-/blob/rivet-3.1.7/configure.ac#L3